python/paddle/fluid/tests/unittests/mkldnn/test_matmul_v2_mkldnn_op.py

```diff
@@ -0,0 +1,284 @@
+        self.y_shape = (1, 1, 100, 2)
+        self.trans_x = False
+        self.trans_y = False
+
+
+class TestMatMulV2MatrixXMatrixTranposeXOneDNNOp2(
+        TestMatMulV2VectorXVectorOneDNNOp):
+    def config(self):
+        self.x_shape = (2, 1, 4, 25)
+        self.y_shape = (1, 1, 4, 25)
+        self.trans_x = True
+        self.trans_y = False
+
+
+class TestMatMulV2MatrixXMatrixTranposeX2OneDNNOp3(
+        TestMatMulV2VectorXVectorOneDNNOp):
+    def config(self):
+        self.x_shape = (2, 2, 5, 4)
+        self.y_shape = (2, 2, 5, 3)
+        self.trans_x = True
+        self.trans_y = False
+
+
+class TestMatMulV2MatrixXMatrixTransposeX3OneDNNOp(
+        TestMatMulV2VectorXVectorOneDNNOp):
+    def config(self):
+        self.x_shape = (3, 1, 6, 5)
+        self.y_shape = (1, 2, 6, 9)
+        self.trans_x = True
+        self.trans_y = False
+
+
+class TestMatMulV2MatrixXMatrix4OneDNNOp(TestMatMulV2VectorXVectorOneDNNOp):
+    def config(self):
+        self.x_shape = (3, 1, 6, 6)
+        self.y_shape = (1, 2, 6, 9)
+        self.trans_x = False
+        self.trans_y = False
+
+
+class TestMatMulV2VectorXMatrix5DOneDNNOp(TestMatMulV2VectorXVectorOneDNNOp):
+    def config(self):
+        self.x_shape = (100)
+        self.y_shape = (1, 2, 2, 100, 2)
+        self.trans_x = False
+        self.trans_y = False
+
+
+class TestMatMulV2Matrix3DXVectorOneDNNOp(TestMatMulV2VectorXVectorOneDNNOp):
+    def config(self):
+        self.x_shape = (2, 1, 40)
+        self.y_shape = (40)
+        self.trans_x = False
+        self.trans_y = False
+
+
+class TestMatMulV2MatrixXMatrixTransposeXTransposeYOneDNNOp(
+        TestMatMulV2VectorXVectorOneDNNOp):
+    def config(self):
+        self.x_shape = (3, 1, 10, 8)
+        self.y_shape = (1, 2, 9, 10)
+        self.trans_x = True
+        self.trans_y = True
+
+
+class TestMatMulV2MatrixXMatrixTransposeY2OneDNNOp(
+        TestMatMulV2VectorXVectorOneDNNOp):
+    def config(self):
+        self.x_shape = (3, 1, 10, 10)
+        self.y_shape = (1, 2, 9, 10)
+        self.trans_x = False
+        self.trans_y = True
+
+
+class TestMatMulV2MatrixXMatrix5DTranposeYOneDNNOp(
+        TestMatMulV2VectorXVectorOneDNNOp):
+    def config(self):
+        self.x_shape = (1, 3, 1, 10, 10)
+        self.y_shape = (3, 1, 2, 9, 10)
+        self.trans_x = False
+        self.trans_y = True
+
+
+#   BF16 TESTS
+def create_bf16_test_class(parent):
+    class TestMatMulV2Bf16OneDNNOp(parent):
+        def set_inputs(self, x, y):
+            self.inputs = {
+                'X': convert_float_to_uint16(x),
+                'Y': convert_float_to_uint16(y)
+            }
+
+        def set_dtype_attr(self):
+            self.attrs['mkldnn_data_type'] = "bfloat16"
+
+        def test_check_output(self):
+            if core.supports_bfloat16():
+                self.check_output_with_place(core.CPUPlace())
+            else:
+                print("Core doesn't support bf16, skipping UT" +
```

Comment:
Print output can be taken over by test engine, maybe better would be sth like i.e.:

> else:
> self.skipTest('core doesn't support bf16')
>

Reply:
Thank you, I haven't noticed that function, I'll change that